js/app/components/login/login.tsx

```diff
@@ -141,6 +144,7 @@ export default function Login() {
               </Text>
               <Input
                 id="pdsUrl"
+                placeholder="alice.bsky.team"
```

Comment:
not `bsky.social`? concerned someone might type in `someuser.bsky.team` in response to this
```suggestion
                placeholder="example.bsky.social"
```

Reply:
oh good point haha. shyed away initially because alice.bsky.social (which most ppl use as a placeholder) is taken lol